OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -31,12 +30,21 @@ public CaptureProperties(ScriptContext context, Actor self)
 		[Desc("Captures the target actor.")]
 		public void Capture(Actor target)
 		{
-			var targetManager = target.TraitOrDefault<CaptureManager>();
-			if (targetManager == null || !targetManager.CanBeTargetedBy(target, Self, captureManager))
-				throw new LuaException($"Actor '{Self}' cannot capture actor '{target}'!");
+			if (!CanCapture(target))
+			{
+				Log.Write("lua", "Actor '{0}' cannot capture actor '{1}'!", Self, target);
```

Comment:
Why are you undoing the string inlining?

Reply:
Ah that is by accident.